bug_update.php

```diff
@@ -169,7 +169,13 @@
 if( ( $t_resolve_issue || $t_close_issue ) &&
 	!relationship_can_resolve_bug( $f_bug_id )
 ) {
-	trigger_error( ERROR_BUG_RESOLVE_DEPENDANTS_BLOCKING, ERROR );
+
+	# this may be overridden with configuration to allow parents to close regardless of
+	# the status of any children
+	if ( OFF === config_get( 'allow_parent_of_unresolved_to_close' ) ) {
```

Comment:
For the sake of consistency, we use loose comparison `OFF == config_get( ...` for such kind of comparison (at least 99% of the code)